configs/components/openssl.rb

```diff
@@ -11,8 +11,9 @@
     pkg.build_requires "pl-gcc-#{platform.architecture}"
     pkg.build_requires 'runtime' if platform.is_huaweios?
     # needed for the makedepend command
+    pkg.build_requires 'imake' if platform.name =~ /^el/
```

Comment:
Not sure if this is useful, but [this thread](https://bugzilla.redhat.com/show_bug.cgi?id=890712) might be tangentially related to the `imake` question.